RealmMapView/RealmMapView.swift

```diff
@@ -76,11 +76,21 @@ public class RealmMapView: MKMapView {
     ///
     /// Default is 20, which means clustering will occur at every zoom level if clusterAnnotations is YES
     public var maxZoomLevelForClustering: ABFZoomLevel = 20
+  
+    //Represent the state of the bubble for the Annotation
+    //
+    //Default is enabled
+    public var canShowBubble : Bool = true
```

Comment:
would prefer this method mirror `canShowCallout` instead of using the word "bubble"